ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/getridof/AbstractStep.java

```diff
@@ -1,4 +1,4 @@
-package io.syndesis.qe.pages.integrations.edit.steps;
+package io.syndesis.qe.pages.integrations.editor.add.steps.getridof;
```

Comment:
Do we want to get rid of getridof package?

Reply:
yes of horse! ...in the next PR